src/js/pm-image-editor.js

```diff
@@ -455,6 +474,16 @@
 
                 $scope.updateEditorCss();
             });
+
+            var watchSelectionDimensions = function(value) {
```

Comment:
The name `watchSelectionDimensions` implies this function will cause the selection dimensions to be watched -- actually, that's done elsewhere (in `$scope.$watch`) -- this function is just the handler for the change event.